src/jdk.compiler/share/classes/com/sun/source/util/DocTreeScanner.java

```diff
@@ -35,6 +35,7 @@
  * Inside your method, call super.visitXYZ to visit descendant
  * nodes.
  *
+ * @implSpec
```

Comment:
1. Should the error-counting example in the immediately following paragraph belong to this `@implSpec` section?
2. Why is there no similar `@implSpec` section being added to the class-level doc comment of TreeScanner? If such a section is to be added, then should the identifier-counting example in the immediately following paragraph belong to that section?

Reply:
Thanks for the comment, Pavel. I've tried to do the changes in an update:
https://github.com/openjdk/jdk/pull/4223/commits/5c9f694fade7380fa1a7ae667cf4acd5be60c244